### PR TITLE
Add upgrade plan options to cluster upgrade docs

### PR DESCRIPTION
**Pull request by chrisnegus**
*Issue #, if available:*
Documentation of [Issue #499](https://github.com/aws/eks-anywhere/issues/499) 

*Description of changes:*

* Show example of `eksctl anywhere upgrade plan` options and output.
* Add Cilium to list of upgraded components.

### CODE REVIEW

File: docs/content/en/docs/tasks/cluster/cluster-upgrades.md [hunk 1]
```diff
@@ -46,6 +47,30 @@ version.
...
+NAME                     CURRENT VERSION                 NEXT VERSION
+EKS-A                    v0.0.0-dev+build.1000+9886ba8   v0.0.0-dev+build.1105+46598cb
+cluster-api              v1.0.2+e8c48f5                  v1.0.2+1274316
+kubeadm                  v1.0.2+92c6d7e                  v1.0.2+aa1a03a
+vsphere                  v1.0.1+efb002c                  v1.0.1+ef26ac1
+kubadm                   v1.0.2+f002eae                  v1.0.2+f443dcf
+etcdadm-bootstrap        v1.0.2-rc3+54dcc82              v1.0.0-rc3+df07114
+etcdadm-controller       v1.0.2-rc3+a817792              v1.0.0-rc3+a310516
+```
+To format output in json, add `-o json | jq` to the end of the command line.
```
Comment (g-gaston) [inline docs/content/en/docs/tasks/cluster/cluster-upgrades.md:72]:
```suggestion
To format output in json, add `-o json` to the end of the command line.
```

I would remove the `jq`. That's only needed if you want to see the json output pretty-formatted or if you want to do some kind of search/manipulation in the output

**Review by g-gaston — APPROVED** [review]

File: docs/content/en/docs/tasks/cluster/cluster-upgrades.md (hunk 1 above)
Comment (chrisnegus) [reply docs/content/en/docs/tasks/cluster/cluster-upgrades.md]: Done.

File: docs/content/en/docs/tasks/cluster/cluster-upgrades.md [hunk 2]
```diff
@@ -46,6 +47,30 @@ version.
...
+Before you perform an upgrade, check the current and new versions of components that are ready to upgrade by typing:
+
+```bash
+eksctl anywhere upgrade plan cluster -f cluster.yaml
+```
+The output should appear similar to the following:
+
+```
+Worker node group name not specified. Defaulting name to md-0.
+Warning: The recommended number fo control plane nodes is 3 or 5
```
Comment (rothgar) [inline docs/content/en/docs/tasks/cluster/cluster-upgrades.md:60]: Is "fo" a bug in the code or just in our example output?

File: docs/content/en/docs/tasks/cluster/cluster-upgrades.md [hunk 3]
```diff
@@ -46,6 +47,30 @@ version.
...
+NAME                     CURRENT VERSION                 NEXT VERSION
+EKS-A                    v0.0.0-dev+build.1000+9886ba8   v0.0.0-dev+build.1105+46598cb
+cluster-api              v1.0.2+e8c48f5                  v1.0.2+1274316
+kubeadm                  v1.0.2+92c6d7e                  v1.0.2+aa1a03a
+vsphere                  v1.0.1+efb002c                  v1.0.1+ef26ac1
+kubadm                   v1.0.2+f002eae                  v1.0.2+f443dcf
+etcdadm-bootstrap        v1.0.2-rc3+54dcc82              v1.0.0-rc3+df07114
+etcdadm-controller       v1.0.2-rc3+a817792              v1.0.0-rc3+a310516
+```
+To format output in json, add `-o json` to the end of the command line.
```
Comment (rothgar) [inline docs/content/en/docs/tasks/cluster/cluster-upgrades.md:72]:
```suggestion
To format output the in json, add `-o json` to the end of the command line.
```

File: docs/content/en/docs/tasks/cluster/cluster-upgrades.md (hunk 2 above)
Comment (chrisnegus) [reply docs/content/en/docs/tasks/cluster/cluster-upgrades.md:60]: That was me. Fixed.

File: docs/content/en/docs/tasks/cluster/cluster-upgrades.md (hunk 3 above)
Comment (chrisnegus) [reply docs/content/en/docs/tasks/cluster/cluster-upgrades.md:72]: Done.

**Review by chrisnegus — COMMENTED** [review]
I think I got all the comments. Thanks @rothgar and @g-gaston

**Pull request merged**